src/types/onyx/PolicyTag.ts

```diff
@@ -7,7 +7,13 @@ type PolicyTag = {
 
     /** "General Ledger code" that corresponds to this tag in an accounting system. Similar to an ID. */
     // eslint-disable-next-line @typescript-eslint/naming-convention
-    'GL Code': string;
+    'GL Code'?: string;
+
+    /** List of tags */
+    tags: PolicyTags;
```

Comment:
I'm confused, why are we adding this here? Why/how does a tag have a list of tags? 😕

Reply:
<img width="796" alt="image" src="https://github.com/Expensify/App/assets/25230417/51c5e261-2970-4976-bebd-61f6976533f1">

this is a model which we have in Onyx

Reply:
Ok, assuming we are talking about the object in the screenshot with `name: "Tag"` and `required: true` and `tags: {...}`, I have some comments to make about `src/types/onyx/PolicyTag.ts`:
- This object does not have a `GL Code` AFAIK
- This object is called a TagList, so it should at the very least be renamed to `PolicyTagList`, and rename this `tags` attribute on list 13 to `tagList` or `tagObjects`

cc @danieldoglas @yuwenmemon

Reply:
Huh, we have a `PolicyTagList` below, so this does not make any sense.

Reply:
You are right `tags` shouldnt exists on single `PolicyTag` that was a mistake,
So now `PolicyTag` is a type for a single tag in `tags` list and `PolicyTagList` is a represantation of a model in Onyx which includes `tags` and `GL Code` is an optional field which comes from some integration with accounting system I guess based on the `proptypes` which was existing before 
<img width="1069" alt="image" src="https://github.com/Expensify/App/assets/25230417/d4b7f607-ac4f-4401-8f97-b5b3eacaf56b">
and README
<img width="1411" alt="image" src="https://github.com/Expensify/App/assets/25230417/f91c98cc-8dbf-47a9-aeb0-f619f2b0f8c0">

Reply:
`GL Code` is part of `Tag`, not a `TagList`